airbyte-integrations/connectors/source-jira/source_jira/streams.py

```diff
@@ -24,6 +29,35 @@
 API_VERSION = 3
 
 
+class JiraErrorHandler(HttpStatusErrorHandler):
+    def __init__(
+        self,
+        stream_name: str,
+        ignore_status_codes: List[int],
+        logger: logging.Logger,
+        error_mapping: Optional[Mapping[Union[int, str, type[Exception]], ErrorResolution]] = None,
+        max_retries: int = 5,
+        max_time: timedelta = timedelta(seconds=600),
+    ) -> None:
+        super().__init__(logger, error_mapping, max_retries, max_time)
+        self.stream_name = stream_name
+        self.ignore_status_codes = ignore_status_codes
+
+    def interpret_response(self, response_or_exception: Optional[Union[requests.Response, Exception]] = None) -> ErrorResolution:
+        # override default error mapping
+        if response_or_exception.status_code in self.ignore_status_codes:
```

Comment:
It seems like this would raise an unexpected exception if `response_or_exception` is of type `Exception`. Should we handle this?

Reply:
Thanks, updated